### DISCUSSION TITLE
primeng Calender not working proper

### DISCUSSION THREAD

**Discussion by bhuvneshgrapes** [primefaces/community#2419] [PrimeNG] [answered]
![image](https://github.com/user-attachments/assets/35b07e2a-db47-4ef2-9eee-9c0265804f1e)
I use PrimeNG Datetime calendar. The DateTime calendar automatically closes after selecting a date. How do I select a time? It takes the current time because the modal closes after selecting a date.

**Comment by ibrahimkelly** [comment, marked as answer]
> ![image](https://private-user-images.githubusercontent.com/89394302/353368210-35b07e2a-db47-4ef2-9eee-9c0265804f1e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjI4NTg3MzcsIm5iZiI6MTcyMjg1ODQzNywicGF0aCI6Ii84OTM5NDMwMi8zNTMzNjgyMTAtMzViMDdlMmEtZGI0Ny00ZWYyLTllZWUtOWMwMjY1ODA0ZjFlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MDUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODA1VDExNDcxN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWFlNmNhNjVlNmMzYjM4OWRhZjAxZDE5MWVjYjg4NWE0OTI1NzBiODk3NmQ4MjY2YjYwMzU3ZDY1ZjI4OTZhMDgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.f8zzHZ4MMx7vKtHyC-i-77JHUIMVH_VwwSunYkaP1F4) I use PrimeNG Datetime calendar. The DateTime calendar automatically closes after selecting a date. How do I select a time? It takes the current time because the modal closes after selecting a date.

As expected, you must select the time first, then the date.
An alternative will be to add ok button to confirm the selection (PrimeNG side)
